install_requirements.sh

```diff
@@ -36,9 +36,11 @@ pip3 install -r requirements.txt
 echo "Installing tflite for x86 CPU"
 if python3 --version | grep -q 3.8
 then
+  # pip3 install --force-reinstall https://github.com/google-coral/pycoral/releases/download/v1.0.1/tflite_runtime-2.5.0-cp38-cp38-linux_x86_64.whl
   pip3 install --force-reinstall https://github.com/google-coral/pycoral/releases/download/v1.0.1/tflite_runtime-2.5.0-cp38-cp38-linux_x86_64.whl
 else
-  pip3 install https://dl.google.com/coral/python/tflite_runtime-1.14.0-cp37-cp37m-linux_x86_64.whl
+  # pip3 install https://dl.google.com/coral/python/tflite_runtime-1.14.0-cp37-cp37m-linux_x86_64.whl
+  pip3 install --force-reinstall https://github.com/google-coral/pycoral/releases/download/v1.0.1/tflite_runtime-2.5.0-cp37-cp37m-linux_x86_64.whl
```

Comment:
@bhavikapanara what is the purpose of reinstalling tflite for python 3.7 immediately after installing it for 3.8 in the line above?

Also, notice that the TFLite team introduced a multi-platform install package, that is capable of automatically picking up the host platform. There is no longer need to script the platform detection code and install the correct binary. See example [here](https://www.tensorflow.org/lite/guide/python).

```
echo "deb https://packages.cloud.google.com/apt coral-edgetpu-stable main" | sudo tee /etc/apt/sources.list.d/coral-edgetpu.list
curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
sudo apt-get update
sudo apt-get install python3-tflite-runtime
```

Reply:
Done

@ivelin Above mention commands are useful for Debian Linux. But, we have used Ubuntu OS, so I have used this command from [document](https://www.tensorflow.org/lite/guide/python).

`pip3 install --index-url https://google-coral.github.io/py-repo/ tflite_runtime`

Reply:
Very cool. I didn't know about it.